contrib/systemd/crio-shutdown.service

```diff
@@ -2,13 +2,11 @@
 Description=Shutdown CRI-O containers before shutting down the system
 Wants=crio.service
 After=crio.service
-Documentation=man:crio(8)
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/rm -f /var/lib/crio/crio.shutdown
-ExecStop=/usr/bin/env bash -c "/usr/bin/mkdir /var/lib/crio; /usr/bin/touch /var/lib/crio/crio.shutdown"
 RemainAfterExit=yes
+ExecStop=/usr/local/bin/crictl rm -af
```

Comment:
@haircommander @mrunalp @umohnani8 @saschagrunert could we have a wrapper CLI as direct call to `stopAllPodSandboxes()` here, similar as `crio wipe` shorthand?

```
ExecStart=/usr/local/bin/crio \
          $CRIO_CONFIG_OPTIONS \
          $CRIO_RUNTIME_OPTIONS \
          $CRIO_STORAGE_OPTIONS \
          $CRIO_NETWORK_OPTIONS \
          $CRIO_METRICS_OPTIONS \
          shutdown
```

Reply:
I would be more comfortable with that. can you spin up a PR

Reply:
> I would be more comfortable with that. can you spin up a PR

I try to keep this PR clean and focus on systemd related issues and temporary handle shutdown with `crio rm -af`.

The implementation of `crio shutdown` goes to #4182; BTW, I have zero knowledge with golang... @haircommander please kindly advice how could we implement it ;-)